lnwallet/payment_descriptor.go

```diff
@@ -42,6 +42,12 @@ const (
 	// FeeUpdate is an update type sent by the channel initiator that
 	// updates the fee rate used when signing the commitment transaction.
 	FeeUpdate
+
+	// NoOpAdd is an update type that adds a new HTLC entry into the log.
+	// This differs from the normal Add type, in that when settled the
+	// balance will go back to the sender, rather than be credited for the
```

Comment:
We should make it clear that this balance goes back conditionally based on the reserve.